tests/test_placeholder.py

```diff
@@ -0,0 +1 @@
+"""Python tests can be added here."""
```

Comment:
```suggestion
"""Python tests can be added here."""
def test_placeholder():
    pass
```

Note, however, that this may be considered an anti-pattern since we are pretending to add tests while in reality we are not.

Reply:
> IMO we should have the ability to specify whether or not the Python component includes tests

That's debatable, and I guess depends on our coding standards @AlidaOdendaal.

As it stands, https://github.com/people-os/process/blob/master/process/Python_Coding_Guide.md#testing goes over unit testing, and so I'd imagine it's fair to expect some coverage.
Otherwise, that same argument could be applied to (not) having docstrings or code style conventions, as the code is still functional without all of that.

The CI effectively nudges users towards adding tests, which is a good practice, and it's a good software engineering pattern IMHO. Once you or somebody else is ready to add unit testing, now it will be slightly easier because there is already one placeholder test in place ✌️ 

Just my 2 cents.